src/util/reactHelpers.ts

```diff
@@ -0,0 +1,16 @@
+import { isValidElement, ReactNode } from 'react';
+
+/**
+ * This utility function checks whether an unkown value is of type ReactNode, useful when
+ * sending notifications from a try/catch
+ */
+export function isReactRenderable(value: unknown): value is ReactNode {
```

Comment:
See above
```suggestion
export function isReactNode(value: unknown): value is ReactNode {
```

Also, what's going on with the return value for the function here? Why not `boolean`?

Seems like `is` should be used for type testing? I.e. maybe in the function?

Reply:
This is to tell TypeScript that the value we passed in is of the type ReactNode
Otherwise we would get an error here: 
```javascript
        if (isReactRenderable(e)) {
          showNotification('Error fetching OpenSpace API', e, LogLevel.Error);  //<--- argument of type 'unknown' is not assignable to parameter of type 'ReactNode'.ts(2345)
        }
```

Reply:
Dumb Typescript question and not a suggestion. How is this different from:
`export function isReactNode(value: ReactNode)`
?

Reply:
You wouldn't be able to call it when the value is unknown which is the case in try/catch blocks. So the following wouldn't work: 

```javascript
try {
}
catch(e) {
 isReactNode(e) // <-- Argument of type 'unknown' is not assignable to parameter of type 'ReactNode'.ts(2345)
} 
```

Reply:
To clarify my previous comment, when using the `value is ReactNode`, TypeScript knows that in the scope of the function where it's called, after that function has finished and returned `true`, the value can be treated as a `ReactNode`